### PR TITLE
fix: apply filters on count of exceptions

### PR DESCRIPTION
**Pull request by makeavish**

### CODE REVIEW

File: pkg/query-service/app/clickhouseReader/reader.go [hunk 1]
```diff
@@ -2593,11 +2593,11 @@ func (r *ClickHouseReader) CountErrors(ctx context.Context, queryParams *model.C
 	args := []interface{}{clickhouse.Named("timestampL", strconv.FormatInt(queryParams.Start.UnixNano(), 10)), clickhouse.Named("timestampU", strconv.FormatInt(queryParams.End.UnixNano(), 10))}
 	if len(queryParams.ServiceName) != 0 {
```
Comment (ankitnayan) [inline pkg/query-service/app/clickhouseReader/reader.go:2594]: @makeavish how was this working earlier as I see `serviceName` is now added to `CountErrorsParams` struct

Comment (makeavish) [reply pkg/query-service/app/clickhouseReader/reader.go:2594]: It wasn't working. i.e. filters weren't applied at all.
Initially, I started on this but later due to context switch missed this scenario.

**Review by palashgdev — APPROVED** [review]
LGTM

**Review by palashgdev — DISMISSED** [review]

File: pkg/query-service/app/clickhouseReader/reader.go [hunk 2]
```diff
@@ -2593,11 +2593,11 @@ func (r *ClickHouseReader) CountErrors(ctx context.Context, queryParams *model.C
...
 		query = query + " AND serviceName = @serviceName"
-		args = append(args, clickhouse.Named("serviceName", queryParams.ServiceName))
+		args = append(args, clickhouse.Named("serviceName", "%"+queryParams.ServiceName+"%"))
 	}
 	if len(queryParams.ExceptionType) != 0 {
 		query = query + " AND exceptionType = @exceptionType"
-		args = append(args, clickhouse.Named("exceptionType", queryParams.ExceptionType))
+		args = append(args, clickhouse.Named("exceptionType", "%"+queryParams.ExceptionType+"%"))
```
Comment (srikanthccv) [inline pkg/query-service/app/clickhouseReader/reader.go:2600]: What does '%' do here? Does it make it pattern matching?

Comment (makeavish) [reply pkg/query-service/app/clickhouseReader/reader.go:2600]: Yes we are pattern matching if serviceName or exceptionType matches any chars

**Review by srikanthccv — APPROVED** [review]

**Review by palashgdev — APPROVED** [review]

**Pull request merged**